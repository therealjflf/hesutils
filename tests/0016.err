
Configuration file: empty
Passwd file:        empty
Group file:         empty

Loading the configuration file
Checking the configuration file

Processing the passwd file entries: 
  --> 0 Hesiod users
Processing the group file entries: 
  --> 0 Hesiod groups
