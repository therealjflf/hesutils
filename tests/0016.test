# Test that the -v option works

CMDLINE=( hesgen -v -d -c empty -p empty -g empty )
RETVAL=0
OUTFILE=0016.out
ERRFILE=0016.err
