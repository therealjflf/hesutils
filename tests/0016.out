local=/hesiod/
local=/ns.athena.mit.edu/

# Users

# Groups

# Group lists

